### ISSUE TITLE
error collectInfo.sh

### ISSUE THREAD

**Issue by liaguiro** [amarinderthind/decontaminer#5]
Hi 

I'm getting empty files after running collectInfo.sh. I think it is related to mineMicrobialData.pl. 
Can you please advise on how to fix this error?

Thanks,

Lina

 -- COLLECTING INFO  --

INFO:   organism type flag -V not specified. O(thers) assumed  (default option).

- PROCESSING DETAILS
INFO: processing NON virus data
INFO: filtering with DEFAULT match count threshold 5.
Can't open input file: No such file or directory at /decontaminer_out/decontaMiner_1.4/perl_scripts/mineMicrobialData.pl line 44.
Can't open input file: No such file or directory at /decontaMiner_1.4/perl_scripts/mineMicrobialData.pl line 44.
INFO: filtering terminated. Generating data Matrix

 WARNING: No output was written Input files do not contain non-zero matches

 WARNING: No output was written Input files do not contain non-zero matches
INFO: processing terminated. Generating HTML report

 ERROR !! NO FILES IN THE DIRECTORY: /decontaminer_out/RESULTS/BACTERIA/COLLECTED_INFO
 ERROR !! NO FILES IN THE DIRECTORY: /decontaminer_out/RESULTS/BACTERIA/COLLECTED_INFOINFO: processing terminated. Output files written in the dir decontaminer_out/RESULTS/BACTERIA/HTML_REPORTS

**Comment by amarinderthind** [comment]
Hi, there. wondering if got an empty directory (o/p) after filterBlastInfo.sh ?

**Comment by liaguiro** [comment]
No. I'm not getting empty directories or files after running filterBlastInfo.sh (for both XXX_alignments.table and
XXX_stats.txt files)

**Comment by amarinderthind** [comment]
Could you send me " a couple of rows of intermediate files and input files"? Is it paired-end data? and please share the command that you used.  Thanks.
